tests/integration/llm/client.py

```diff
@@ -772,6 +772,14 @@ def get_model_name():
     }
 }
 
+handler_performance_model_spec = {
+    "tiny-llama-model": {
+        "batch_size": [1, 128],
```

Comment:
batch size is a bit small, we can go even larger like 512 with tiny llama. This could help to zoom the perf diff